src/utils/inactiveUsers/index.js

```diff
@@ -0,0 +1,27 @@
+const {
```

Comment:
this file seems redundant and its better to just inline these 2 functions

Reply:
Removed one-liners and just added their contents into corresponding places.